app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -48,6 +48,7 @@ class Settings private constructor(
         const val trackingProtectionOnboardingMaximumCount = 2
         const val FENIX_PREFERENCES = "fenix_preferences"
 
+        var overrideTPPopupsForPerformanceTest = false
```

Comment:
afaict, this probably shouldn't be in the companion object: static state is generally hard to reason about and this object has no precedence for managing such state.